### PR TITLE
docs: improve README.md

### PR DESCRIPTION
**Pull request by Iffabled**
`README.md` is the file newcomers see when they first visit the module's repository or NPM page. Currently, it gives absolutely no information about how the module works. This pull request aims to improve by adding information about features, setup and basic usage of this module.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,53 +114,68 @@ git clone git@github.com:<username>/nuxt-i18n.git modules/nuxt-i18n
...
 
-- Install the dependencies
+- Install the dependencies:
 
 ```sh
-cd nuxt-i18n/
-yarn
+cd nuxt-i18n
+yarn # yarn
+npm i # npm
```
Comment (rchl) [inline README.md]: Maybe let's skip whole clone/install deps steps for `Documentation` header since those are already listed for the project above?

**Comment by rchl** [comment]
I like the changes. Thanks. :)

File: README.md (hunk 1 above)
Comment (Iffabled) [reply README.md]: Then it would be unclear which of these steps apply to contributing to documentation and which don't.

Comment (rchl) [reply README.md]: But it's all the same repo. There is no difference in contributing to code or documentation really, apart from running `docs:dev` script to develop documentation.

But if you are insisting, I won't hold it against this review. It's not that important.

If keeping it, there is a bit of incosistency as it mentions `npm` step but other places only mention `yarn`.

**Comment by rchl** [comment]
Thank you.

**Pull request merged**